plugins-mu/hm-stack/tests/bootstrap.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Bootstrap and load test files.
+ *
+ * @package HMStackIntegration
+ * @author Human Made Limited
+ */
+
+// Test functionality as if on production.
+define( 'HM_DEV', false );
+
+// Setup Variables for testing.
+define( 'HM_STACK_API_URL', 'https://eu-west-1.aws.hmn.md/api/stack/applications/test-production/' );
```

Comment:
I'd rather the unit tests not be running against this, or maybe I'm misunderstandingit?

Reply:
Ah I see it's stubbed, though I think it would be better to not have it potentially run through to this URL. I'd use a fake domain.